gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
> logically, this is to test the gem, not your local cli.rb file.

no, the point is to test the on disk version of the code & not the installed gem version.

we only ever want to test the local version, why use require?